### ISSUE TITLE
Intellisense variable description inconsistency.

### ISSUE THREAD

**Issue by tanzanite777** [DEVSENSE/phptools-docs#463]
Type: <b>Feature Request</b>

Intellisense does not show variable description OR main documentation block when hovered depending on place of usage of the keyword @var.

```
class Foo {
  /**
   * Main documentation block is shown.
   * @var int Description is not shown.
   */
  int $snafu1;

  function snafu2() {
    /**
     * Main documentation block is not shown.
     * @var int Description is shown.
     */
    int $snafu2;
  }

  /** @var int Typical usage that breaks when working with VSCode + PHP as none of this will actually show up anywhere. */
  int $issueExample;

}
```

Should never omit any of them. Did not find how exactly phpdoc expects it to be documented (besides just showing that both are valid), but IDE does not need to adhere to that anyway.

My suggestion is to show text from whichever place is present (joining the texts with empty line between if both are present).

Should be trivial change - but much appreciated.

Extension version: 1.42.14434
VS Code version: Code 1.85.1 (0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2, 2023-12-13T09:49:37.021Z)
OS version: Windows_NT x64 10.0.19045
Modes:

**Comment by jakubmisek** [comment]
thank you for the suggestion - will be implemented!

**Comment by jakubmisek** [comment]
implemented in `v1.42.14618`, thank you!
